### PR TITLE
Add `NationalPost`

### PR DESCRIPTION
**Pull request by addie9800**

### CODE REVIEW

File: src/fundus/publishers/ca/national_post.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            preliminary_topics = self.precomputed.ld.bf_search("keywords")
+            filter_list = ["Curated", "News", "Newsroom daily", "story", "Canada", "World"]
+            filtered_topics = [
+                topic
+                for topic in preliminary_topics
+                if "NLP Entity Tokens" not in topic
+                and "NLP Category" not in topic
+                and topic not in filter_list
+                and not re.search(r"[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}", topic)
+            ]
```
Comment (MaxDall) [inline src/fundus/publishers/ca/national_post.py:61]: Maybe some sort of regex filter would be nice here. Actually, I think you could just use the regular `regex_filter`.

File: src/fundus/publishers/ca/__init__.py [hunk 2]
```diff
@@ -16,3 +17,14 @@ class CA(metaclass=PublisherGroup):
...
         ],
     )
+
+    NationalPost = Publisher(
+        name="National Post",
+        domain="https://nationalpost.com",
+        parser=NationalPostParser,
+        sources=[
+            NewsMap("https://nationalpost.com/sitemap-news.xml"),
+            Sitemap("https://nationalpost.com/sitemap.xml"),
```
Comment (MaxDall) [inline src/fundus/publishers/ca/__init__.py:27]:
```suggestion
            Sitemap("https://nationalpost.com/sitemap-old.xml"),
```

File: tests/resources/parser/test_data/ca/NationalPost.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "V1": {
+    "authors": [
+      "Special to National Post"
+    ],
+    "body": {
+      "summary": [
+        "A draft policy by the CMA essentially calls for an end to private workplace coverage for virtual health care"
+      ],
+      "sections": [
+        {
```
Comment (MaxDall) [inline tests/resources/parser/test_data/ca/NationalPost.json:11]: I think it would be good to add a test case including at least one subheadline.

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding 👍

File: src/fundus/publishers/ca/national_post.py (hunk 1 above)
Comment (addie9800) [reply src/fundus/publishers/ca/national_post.py]: I do agree. Using regex_filter seems a bit cursed Typing wise, but it does work just fine.

**Review by MaxDall — APPROVED** [review]
Thanks for adding! Looks good 👍

**Pull request merged**